CMakeLists.txt

```
@@ -25,5 +29,19 @@ if(BUILD_TESTING)
 endif()
 
 add_subdirectory(src/beman/exemplar)
-
 add_subdirectory(examples)
+
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_PROJECT_NAME}ConfigVersion.cmake
+  VERSION ${PROJECT_VERSION}
+  COMPATIBILITY AnyNewerVersion)
```

```suggestion
  COMPATIBILITY ExactVersion)
```

Unless we're doing something specific to guarantee some ABI compatibility across versions of a Beman project, we should be conservative about version matching. On the contrary, I fully expect API and ABI breaks across releases of Beman projects since they're by definition experimental and prone to design changes.

We could *maybe* revisit this choice on a case-by-case basis for projects that a "stable" status. But unless they're spelling their implementations with ABI tags and such, I wouldn't advise people bother.